examples/bucket/bin/index.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
```

Comment:
Do we intend to commit the `bin/` directory?

Reply:
Nope!  Will fix.